spec/ttml1.xml

```diff
@@ -13244,6 +13243,53 @@ shown in <specref ref="fragment-streaming-graphic"/>.</p>
 <p>This specification does not define a transport buffer model or a decoder
 capabilities model.</p>
 </note>
+</div2>
+<div2 id="temporal-fragmentation">
+<head>Temporal fragmentation</head>
+<p>Another possible means by which TTML Content may be streamed is to
```

Comment:
Remove "possible".

Reply:
This is the same wording as in the opening text of the root and branch fragmentation section.